deploy/v1.4.0/mainnet.ts

```diff
@@ -0,0 +1,115 @@
+import { dim } from 'chalk';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { deployAndLog } from '../../src/deployAndLog';
+import { setManager } from '../../src/setManager';
+import { transferOwnership } from '../../src/transferOwnership';
+
+/**
+ * @description Updates the PrizeTierHistory with an optimized BinarySearchLib contract.
+ * @dev Due to the immutably linking contracts together, a series of new contracts must be deployed
+ *      in succession and the PrizePool must be updated by the PoolTogether executive team.
+ */
+export default async function deployToEthereumMainnet(hre: HardhatRuntimeEnvironment){
+    if (process.env.DEPLOY === 'v1.4.0.mainnet') {
+        dim(`Deploying: PrizeTierHistory Ethereum Mainnet`)
+        dim(`Version: 1.4.0`)
+    } else { return }
+
+    const { deployer, executiveTeam, ptOperations, defenderRelayer } = await hre.getNamedAccounts();
+    const drawCalculator = await hre.ethers.getContract('DrawCalculator');
+
+    const prizeTierHistoryOld = await hre.ethers.getContract('PrizeTierHistory');
+    const lastPrizeTier = await prizeTierHistoryOld.getPrizeTier(await(prizeTierHistoryOld.getNewestDrawId()));
+    
+    /* PrizeTierHistory ------------------ */
+    // @dev Required by PrizeDistributionFactory
+    /* ----------------------------------- */
+    await deployAndLog('PrizeTierHistory', {
+        from: deployer,
+        args: [deployer],
+        skipIfAlreadyDeployed: false,
+    });
+    const prizeTierHistory = await hre.ethers.getContract('PrizeTierHistory');
+    await prizeTierHistory.push(lastPrizeTier)
+
+    /* PrizeDistributionFactory ---------- */
+    // @dev Immutable reference to new PrizeTierHistory
+    // @dev Use existing references for the remaining contracts
+    // @dev Required by BeaconTimelockTrigger
+    /* ----------------------------------- */
+    const drawBuffer = await hre.ethers.getContract('DrawBuffer');
+    const prizeDistributionBuffer = await hre.ethers.getContract('PrizeDistributionBuffer');
+    const ticket = await hre.ethers.getContract('Ticket');
+    const minPickCost = 1000000
```

Comment:
We could use the constant `PRIZE_DISTRIBUTION_FACTORY_MINIMUM_PICK_COST` from `src/constants.ts`.